src/permission-set.js

```diff
@@ -16,523 +16,809 @@
  *     as 'to be inherited', that is will have `acl:default` set.
  */
 
-var Authorization = require('./authorization')
-var acl = Authorization.acl
-var ns = require('solid-namespace')
+const Authorization = require('./authorization')
```

Comment:
The reason I see is that they're standardized/accepted in the ES Standard (TM), whereas `require` is a convention provided by `npm` and adopted by the community before modules had been standardized in the language.